weave/index/index_sqlite.py

```diff
@@ -323,7 +327,7 @@ def get_parents(self, basket_address, **kwargs):
         columns.append("path")
 
         parent_df = pd.DataFrame(self.cur.execute(
-            """WITH RECURSIVE
+            f"""WITH RECURSIVE
```

Comment:
f strings can be fairly dangerous in SQL statements, as they can expose the query to injection attacks. Only use f strings in these queries if the values being replaced are predetermined and not input by the user.
Remove the f string, and add an additional argument on like 347: ```... (basket_uuid, basket_uuid, max_gen_level)``` then replace "{max_gen_level}" on line 342 with "?". 
Apply to get_children as well.